spec/factory_girl/registry_spec.rb

```diff
@@ -57,4 +57,19 @@
     subject.clear
     expect(subject.count).to be_zero
   end
+
+  it "removes a factory" do
+    subject.register(:object_name, registered_object)
+    expect do
+      expect(subject.remove(:object_name)).to eq(registered_object)
```

Comment:
i guess I figured if `Hash#delete` returned the object deleted there might be a good reason to do that, and I should just pass it on. But no strong opinion.